spatialdata/_core/_spatial_query.py

```diff
@@ -85,3 +89,129 @@ def _bounding_box_query_points_dict(
             requested_points[points_name] = points
 
     return requested_points
+
+
+def _bounding_box_query_image(
+    image: Union[MultiscaleSpatialImage, SpatialImage], request: BoundingBoxRequest
+) -> Union[MultiscaleSpatialImage, SpatialImage]:
+    """Perform a spatial bounding box query on an Image or Labels element.
+
+    Parameters
+    ----------
+    image : Union[MultiscaleSpatialImage, SpatialImage]
+        The image element to perform the query on.
+    request : BoundingBoxRequest
+        The request for the query.
+
+    Returns
+    -------
+    query_result : Union[MultiscaleSpatialImage, SpatialImage]
+        The image contained within the specified bounding box.
+    """
+    spatial_axes = _get_spatial_axes(request.coordinate_system)
+
+    # build the request
+    selection = {}
+    for axis_index, axis_name in enumerate(spatial_axes):
+        # get the min value along the axis
+        min_value = request.min_coordinate[axis_index]
+
+        # get max value, slices are open half interval
+        max_value = request.max_coordinate[axis_index] + 1
+
+        # add the
+        selection[axis_name] = slice(min_value, max_value)
+
+    query_result = image.sel(selection)
+
+    # update the transform
+    # currently, this assumes the existing transforms input coordinate system
+    # is the intrinsic coordinate system
+    # todo: this should be updated when we support multiple transforms
```

Comment:
this whole section will have to be refactored when #100 lands. However, in order to keep things moving, I would like to merge this and refactor later if possible.

Reply:
I agree with you